app/src/app/components/version-popover/version-popover.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { StixObject } from 'src/app/classes/stix/stix-object';
+
+@Component({
+    selector: 'app-version-popover',
+    templateUrl: './version-popover.component.html',
+    styleUrls: ['./version-popover.component.scss']
+})
+export class VersionPopoverComponent implements OnInit {
+    @Input() public config: VersionPopoverConfig;
+
+    public markdownContent: string = '';
+
+    // General guidelines applicable to all objects
+    private generalGuidelines: string = `
+### **General Versioning Guidelines**
+#### **Goals**:
+- ATT&CK moves to incremental updates.
+- Users can track when an item has changed between releases.
+- Users have a reliable indication of the degree of change to an item.
+- Published items remain stable references.
```

Comment:
The goals aren't necessarily version bump guidelines, so this section can be removed